docs/reference/plugins.md

```diff
@@ -0,0 +1,81 @@
+# Plugins
+
+By default, the following WordPress plugins are installed with the latest version during the OCI
+image build time. Hence, if WordPress is deployed at a later stage, there may be version
+differences between the plugins across the WordPress units.
```

Comment:
The discrepancy of plugins between units will only happen if user install additional plugins through the `plugin` configuration. The default plugins came with the docker image will always be the same across all units. But plugins defined by the `plugins` configuration will be installed when ever the plugin is not persisted in the pod. So the plugin defined by the `plugins` configuration can have a different installation time.

Reply:
Updated the documentation to note the information on runtime installed plugins. Thank you for that!